### PR TITLE
fix task 3

### PR DESCRIPTION
**Pull request by Julian-Carlosama**

### CODE REVIEW

File: tests/test_models/test_engine/test_db_storage.py [hunk 1]
```diff
@@ -87,10 +87,21 @@ def test_new(self):
...
+
+    @unittest.skipIf(models.storage_t == 'db', "not testing file storage")
+    def test_count(self):
+        """Test that save properly saves objects to file.json"""
+        n_state = models.storage.count('State')
+        ad_state = State(name="Arizona")
+        models.storage.new(ad_state)
+        models.storage.save()
+        n_state_ad = models.storage.count("State")
+        self.assertEqual(n_state + 1, n_state_ad)
```
Comment (LauraAlejandra2021) [inline tests/test_models/test_engine/test_db_storage.py:116]: you can add new test on get method passing none by class

**Review by LauraAlejandra2021 — APPROVED** [review]
